app/packages/state/src/hooks/useCreateLooker.ts

```diff
@@ -56,7 +56,10 @@ export default <T extends FrameLooker | ImageLooker | VideoLooker>(
 
       const mimeType = getMimeType(sample);
 
-      if (mimeType !== null) {
+      // checking for pcd extension instead of media_type because this also applies for group slices
+      if (urls.filepath.endsWith(".pcd")) {
```

Comment:
This seems a bit contradictory to the plugin "activator" function. Eg. does this mean that plugin devs cannot provide their own pcd visualizer?